questions.md

```diff
@@ -0,0 +1,124 @@
+<p align="center">
+<img src="logo.png">
+</p>
+
+
+babylon health
+==================================
+
+As described in the iOS recruitment process [main page](TODO), the face-to-face technical interviews involve us asking candidates a number of technical questions which we have decided to open-source.
+
+Please keep in mind that this document is in a continuous state of flux, with new questions being added, and old ones tweaked or removed, somewhat periodically.
+
+## GENERAL, CV-related
+
+- Questions related to the information on the candidate's CV
+- What technical books and blogs have you read or are reading at the moment?
+- What conferences have you been to or plan on attending, if any?
+- Questions related to the candidate's open source contributions, if any.
+- How would you explain reactive programming to a junior developer who knows nothing about it?
```

Comment:
Don't think this question makes sense in this category.

Reply:
Please suggest where to put it.

Reply:
I moved it to the Swift category. Fixed in https://github.com/Babylonpartners/iOS-Interview-Demo/pull/4/commits/e7fe1ad6c4f0e5b7611b4ec0de08ba494d1fd033.